#! /bin/sh /usr/share/dpatch/dpatch-run
## dvd.c patch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Removes stderr output on correctly parsed -C parameter

@DPATCH@
Index: vdr-plugin-dvd-0.3.6~b03+cvs20090426.0013/dvd.c
===================================================================
--- vdr-plugin-dvd-0.3.6~b03+cvs20090426.0013.orig/dvd.c	2007-09-16 18:31:50.000000000 +0200
+++ vdr-plugin-dvd-0.3.6~b03+cvs20090426.0013/dvd.c	2011-04-09 17:23:43.000000000 +0200
@@ -73,7 +73,7 @@
 #endif
         switch (c) {
             case 'C':
-                fprintf(stderr, "arg: %s\n", optarg);
+                // fprintf(stderr, "arg: %s\n", optarg);
                 cDVD::SetDeviceName(optarg);
                 if (!cDVD::DriveExists()) {
                     esyslog("vdr: DVD drive not found: %s", optarg);
